### PR TITLE
Feature/add message extension already installed

### PR DESCRIPTION
**Pull request by Manu1400**
https://trello.com/c/URqTzWAU/806-d%C3%A9tecter-si-lutilisateur-a-d%C3%A9j%C3%A0-install%C3%A9-lextension-et-ajouter-un-message-vous-avez-d%C3%A9j%C3%A0-install%C3%A9-lextension

### CODE REVIEW

File: app/components/Utils/AlreadyInstalled.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import { checkExtensionInstall } from '../Utils'
+
+@translate('error')
+export default class AlreadyInstalled extends React.PureComponent {
+  state = {isInstalled: false}
```
Comment (julien-leclercq) [inline app/components/Utils/AlreadyInstalled.jsx]: Is a state really required here ?

File: app/components/Utils/Extension.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Extension content scripts load after CaptainFact. We could have created a message
+ * interface to communicate between the two but as our need is very basic for now
+ * (detecting if extension is installed) we wait 5 seconds and check.
+ * @returns {Promise}
+ */
+export const checkExtensionInstall = () => {
+  return new Promise(fulfill => {
+    setTimeout(() => fulfill(!!document.getElementById('captainfact-extension-installed')), 5000)
```
Comment (Betree) [inline app/components/Utils/Extension.jsx]: `5000` you can play with different settings here. Maybe accept it as a param, or try multiple times (`setInterval`) with a max number of retries

File: app/components/Utils/AlreadyInstalled.jsx (hunk 1 above)
Comment (julien-leclercq) [reply app/components/Utils/AlreadyInstalled.jsx]: after a discussion with @Betree let's let this stateful

**Comment by Betree** [comment]
@julien-leclercq This doesn't look like a priority and I'm still not convinced that this PR serves a meaningful purpose. Plus it still requires some dev.

Can we close this ?

**Pull request closed without merging**